CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+### 1.1.0 - 2018-02-27
+
+* (SERVER-2130) Default to jit compile-mode for jruby 9k
+* (SERVER-2081) Set KCode, source encoding and external encoding to ensure
```

Comment:
I notice we haven't merged the 0.12 release into this branch. Is that something we'll want to do, or for small things like this do we keep the branches separate?

Reply:
I'm cool either way as long as the versions interleave chronologically. merge conflicts in changelogs aren't super pleasant though.